test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -308,9 +303,11 @@ static void sched_connect(grpc_closure* closure, grpc_endpoint** ep,
   fc->closure = closure;
   fc->ep = ep;
   fc->deadline = deadline;
-  grpc_timer_init(
-      &fc->timer, grpc_core::Duration::Seconds(1) + grpc_core::Timestamp::Now(),
-      GRPC_CLOSURE_CREATE(do_connect, fc, grpc_schedule_on_exec_ctx));
+  GetDefaultEventEngine()->RunAfter(grpc_core::Duration::Seconds(1), [fc] {
```

Comment:
I think maybe `fc` can be a local variable now, no allocations required.

```
future_connect fc;
fc.closure = closure;
...
GetDefaultEventEngine()->RunAfter(grpc_core::Duration::Seconds(1), [fc = std::move(fc)] mutable {
    grpc_core::ApplicationCallbackExecCtx callback_exec_ctx;
    grpc_core::ExecCtx exec_ctx;
    do_connect(std::move(fc));
  });
```

Reply:
Great point! Thanks for catching it! Looks like `future_connect` and `addr_req` are trivially-copyable types so no need to use `std::move` here.